handler.go

```diff
@@ -54,8 +54,8 @@ func (h *indexHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 }
 
 type packageHandler struct {
+	pkg       PackageConfig
```

Comment:
nit: the line offset with this struct looks unaligned?

Reply:
That's weird. I don't know how that made it in. Re-ran gofumpt over the code again; only this one was messed up.

Looks like the lint doesn't check for gofmt. I'll add one for that in a follow up.